vfio-ubind/src/main.rs

```diff
@@ -65,14 +64,10 @@ fn main() {
 	let dbdf_regex = Regex::new(r"^[[:xdigit:]]{4}:[[:xdigit:]]{2}:[[:xdigit:]]{2}.[[:xdigit:]]$").unwrap();
 	let bdf_regex = Regex::new(r"^[[:xdigit:]]{2}:[[:xdigit:]]{2}.[[:xdigit:]]$").unwrap();
 	
-	if !dbdf_regex.is_match(&device) {
-		if !bdf_regex.is_match(&device) {
-			println!("Please supply Domain:Bus:Device.Function of PCI device in form: dddd:bb:dd.f");
-			return;
-		} else {
-			warn!("No PCI domain supplied, assuming PCI domain is 0000");
-			device = "0000:".to_string() + &device;
-		}
+	match (dbdf_regex.is_match(&device), bdf_regex.is_match(&device)) {
+		(false, true) => {warn!("No PCI domain supplied, assuming PCI domain is 0000");	device = "0000:".to_string() + &device;},
```

Comment:
Missing space between `{` and `warn`. Also if you open a block, put the content in new lines.